library/HTMLPurifier/HTMLModule/HTML5/Tables.php

```diff
@@ -17,7 +17,12 @@ class HTMLPurifier_HTMLModule_HTML5_Tables extends HTMLPurifier_HTMLModule
      */
     public function setup($config)
     {
-        $this->addElement('caption', false, 'Inline', 'Common');
+        // https://html.spec.whatwg.org/multipage/tables.html#the-caption-element
+        $caption = $this->addElement('caption', false, 'Flow', 'Common');
+        $caption->excludes = $this->makeLookup(
+            // Exclude tables.
```

Comment:
Unnecessary comment here. 

Also, since there is only one element in the lookup table, the whole statement can be compressed to a single line.

Reply:
Should be resolved.